library/games/Wordy/0.json

```diff
@@ -669,6 +667,12 @@
       {
         "func": "TURN"
       }
+    ],
+    "globalUpdateRoutine": [
```

Comment:
This should be something else, right?

Reply:
> This should be something else, right?

Oh good grief.  I meant `gameStartRoutine`.  Fixed.